web/src/components/organisms/Asset/Asset/hooks.ts

```diff
@@ -71,11 +73,17 @@ export default (assetId?: string) => {
 
   useEffect(() => {
     switch (true) {
-      case (selectedPreviewType === "GEO" ||
-        selectedPreviewType === "GEO3D" ||
-        selectedPreviewType === "MODEL3D") &&
-        (fileFormats.includes(assetFileExt) || compressedFileFormats.includes(assetFileExt)):
-        setViewerType("cesium");
+      case selectedPreviewType === "GEO" &&
+        (geoFormats.includes(assetFileExt) || compressedFileFormats.includes(assetFileExt)):
```

Comment:
Can this line be extracted into a new function:
`geoFormats.includes(assetFileExt) || compressedFileFormats.includes(assetFileExt)`
I think it is very redundant in the same file

Reply:
Can you explain more?

Reply:
Anyways, I will do this in a separate PR.